### PR TITLE
removed launchpad references; also made SPI credentials optional

### PR DESCRIPTION
**Pull request by fgimenez**

### CODE REVIEW

File: bin/local-provision.sh [hunk 1]
```diff
@@ -7,18 +7,12 @@ then
 fi
 if [ -z "$2" ]
 then
-    echo "No launchpad credentials path given as second argument, exiting"
-    exit 1
-fi
-if [ -z "$3" ]
-then
-    echo "No snappy product integration credentials path given as third argument, exiting"
+    echo "No snappy product integration credentials path given as second argument, SPI won't be available"
```
Comment (elopio) [inline bin/local-provision.sh]: I liked the other message better:  won't be able to connect to SPI

File: bin/local-provision.sh [hunk 2]
```diff
@@ -7,18 +7,12 @@ then
...
     exit 1
```
Comment (elopio) [inline bin/local-provision.sh]: You forgot to remove this one.

**Comment by elopio** [comment]
+0.5. I left a couple of comments ^.

**Comment by fgimenez** [comment]
Fixed, thanks! :)

**Pull request merged**